docs/usage/golang.md

```diff
@@ -40,11 +40,13 @@ Renovate will commit all files changed within the `vendor/` folder.
 
 ### Go binary version
 
-By default, Renovate will keep up with the very latest version of `go`.
+By default, Renovate will keep up with the latest version of the `go` binary.
 
-You can "pin" the `go` version that Renovate uses.
-Say you want Renovate to use Go version 1.14, you can do this by adding `go 1.14` to your `go.mod` file.
-We do not support pinning Go versions to a specific patch level, so you cannot use `go 1.14.12`, but you can use `go 1.14` in your `go.mod` file.
+You can set a contraint in your `go.mod` file to force Renovate to use a specific version.
+As an example, say you want Renovate to use the latest patch version of the `1.16` Go binary, then you would put `contstraints.go=1.16` in your `go.mod` file.
```

Comment:
Please check if this `contraints` example is correct.

I've bumped the version of Go in the examples to `1.16.6` or `1.16`, as that is the latest binary. 😄

Reply:
This should be renovate config, not go.mod.